apps/actions-dependencies-updater/src/generated/types/index.mts

```diff
@@ -0,0 +1,4 @@
+import { MySchema as WorkflowSchema } from "./github-workflow.mjs";
+import { MySchema as ActionSchema } from "./github-action.mjs";
+
+export { ActionSchema, WorkflowSchema };
```

Comment:
```suggestion
export { MySchema as WorkflowSchema } from "./github-workflow.mjs";
export { MySchema as ActionSchema } from "./github-action.mjs";
```
This _might_ work, I'm a little rusty on export syntax

Reply:
I'll leave it as simple and easy to read - an extra line makes it more obvious what is going on imo.